FEM/rf_bc_new.cpp

```diff
@@ -462,6 +472,34 @@ std::ios::pos_type CBoundaryCondition::Read(std::ifstream* bc_file,
             continue;
         }
 
+        if (line_string.find("$TIME_INTERVAL") != std::string::npos)
+        {
+            in.str(readNonBlankLineFromInputStream(*bc_file));
+            double t1, t2;
+            in >> t1 >> t2;
+            in.clear();
+            _time_intervals.push_back(new BaseLib::TimeInterval(t1, t2));
```

Comment:
Following the convention, I suggest something like this

$TIME_INTERVAL
(the number of intervals)
(t1) (t2) ; for 1st intervals 
(t1) (t2) ; for 2nd intervals 
...

Reply:
Thanks for the nice suggestion. However the input syntax has been used in one of our recent delivered report for other users. I can change it later on with an agreement from others. For this PR. I have to keep it as it is.

Reply:
please change it later. I'm not convinced with that we have to accept this because others are already using it in a branch version.

Reply:
I will change it later on.